apps/server/src/assets/mail/templates/changePw.hbs

```diff
@@ -0,0 +1,41 @@
+<html>
```

Comment:
dateiname bitte in kebab-case (vllt zu `change-password.hbs`) ändern denn findis besser lesbar